src/main/java/core/service/OperationStrategy.java

```diff
@@ -0,0 +1,4 @@
+package core.service;
+
+public interface OperationStrategy {
```

Comment:
strategy is a simple class. The idea behind it - is the ability to provide a respective handler to each operation type. Which data structure sounds suitable for this task? Create a constructor, where you will pass the map, and set it to a field. Than create a method, that would provide handler for an operation type